pkg/controller/account/account_controller.go

```diff
@@ -383,35 +416,53 @@ func (r *ReconcileAccount) Reconcile(request reconcile.Request) (reconcile.Resul
 			return reconcile.Result{}, err
 		}
 
-		// Initialize all supported regions by creating and terminating an instance in each
-		err = r.InitializeSupportedRegions(reqLogger, currentAcctInstance, awsv1alpha1.CoveredRegions, creds)
-		if err != nil {
-			errMsg := fmt.Sprintf("Unable to build or destroy ec2 instances: %s", err)
-			r.setStateFailed(reqLogger, currentAcctInstance, errMsg)
+		// We're about to kick off region init in a goroutine. This status makes subsequent
+		// Reconciles ignore the Account (unless it stays in this state for too long).
+		utils.SetAccountStatus(currentAcctInstance, "Initializing Regions", awsv1alpha1.AccountInitializingRegions, AccountInitializingRegions)
+		if err := r.statusUpdate(reqLogger, currentAcctInstance); err != nil {
```

Comment:
@clcollins Would you please vet that I've done these things according to the new pattern? The PR originated before yours landed.

Reply:
This looks fine here!